COLLABORATOR_GUIDE.md

```diff
@@ -378,6 +379,24 @@ multiple commits. Commit metadata and the reason for the revert should be
 appended. Commit message rules about line length and subsystem can be ignored.
 A Pull Request should be raised and approved like any other change.
 
+### Additions to the Cryptography and Security APIs
+
+Semver-minor commits that add or change cryptograpy APIs and Security APIs
```

Comment:
Does `tls` fall under "security" or "crypto", and if it's "crypto" then are there any "security APIs" at all?